changelog.md

```diff
@@ -4,6 +4,9 @@
 
 ### Added
 
+- Support for qualifying scalar types by their schema. This updates the
+  metadata configuration format version number from `"3"` to `"4"`.
```

Comment:
```suggestion
  metadata configuration format version number from `"3"` to `"4"`.
  ([#471](https://github.com/hasura/ndc-postgres/pull/471))
```

Reply:
Ah, I had already made this change when I saw this. Thanks though!